### DISCUSSION TITLE
PiVPN project interview

### DISCUSSION THREAD

**Discussion by ph4ni** [pivpn/pivpn#1722] [Q&A]
Hi! I'm Sai from [Console](https://console.substack.com/). We curate projects, interview the maintainers, and send them to thousands of our readers each week.

I came across PiVPN from a blog when trying to self-host a VPN in my home network, and it looks really great. I think our readers would also love to know more about it. So, I am reaching out to know if you would be interested in sharing your story and the project in a text-based interview. Let me know what you think!

**Comment by coolapso** [comment]
Hi, not really sure about it, but ... hit me up through a proper channel (not that hard to find) and i will think about it.
